gallery/src/main.scss

```diff
@@ -81,3 +81,38 @@
     display: none;
   }
 }
+
+#temporary-navigation-sidebar-example {
+  .navigation-sidebar-custom-content-1 {
```

Comment:
These elements are much more vertically spaced out in the impl than in the mockup

Reply:
> These elements are much more vertically spaced out in the impl than in the mockup

See my note in the PR description about the mock not being pixel perfect. This is just an example of something that can be accomplished with the component, not something that's ought to be exactly like the guideline.

Reply:
> These elements are much more vertically spaced out in the impl than in the mockup

Updated in 370ebe4de4242e642ff438afc663c443df76dced